lib/adiwg/mdtranslator/writers/dcat_us/sections/dcat_us_temporal.rb

```diff
@@ -8,7 +8,15 @@ def self.build(intObj)
                   resourceInfo = intObj.dig(:metadata, :resourceInfo)
                   extent = resourceInfo&.dig(:extents, 0)
                   temporalExtents = extent&.dig(:temporalExtents)
-                
+
+                  if temporalExtents.nil? || temporalExtents.empty?
```

Comment:
based on jonathan's comments, we should not be modifying this code and instead should be modifying our test fixtures.